### PR TITLE
Build a react calculator app (milestone-four) - Logic

### PR DESCRIPTION
**Pull request by ampaire**
This is the implementation of milestone four of the calculator app in react.

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import Big from 'big-js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const [one, two] = [Big(numberOne), Big(numberTwo)];
+
```
Comment (shloch) [inline src/logic/operate.js:5]: wise one

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  switch (operation) {
+    case '+':
+      return one.plus(two);
+    case '-':
+      return one.minus(two);
+    case 'x':
+      return one.times(two);
+    case '/':
+      return one.div(two);
```
Comment (shloch) [inline src/logic/operate.js]: Dont forget to handle DIVISION by ZERO on milestone 5 else, it will generate an error

**Review by shloch — APPROVED** [review]
## Hello @ampaire 
accomplished

### Great job...good to go

File: src/logic/operate.js (hunk 2 above)
Comment (ampaire) [reply src/logic/operate.js]: Thank you @Shloch. This is a good suggestion.

**Pull request merged**